data-otservbr-global/scripts/actions/store_coins.lua

```diff
@@ -0,0 +1,12 @@
+local storeCoin = Action()
```

Comment:
I believe that not always servers would want this feature. Maybe make this configurable?

Reply:
that's fair. Although this item that we use here is specifically Tibia coins tho. And it's unobtainable